src/sentry/static/sentry/app/routes.jsx

```diff
@@ -42,7 +42,7 @@ import ProjectInstallPlatform from 'app/views/projectInstall/platform';
 import ProjectReleases from 'app/views/projectReleases';
 import ProjectSavedSearches from 'app/views/projectSavedSearches';
 import ProjectSettings from 'app/views/projectSettings';
-import ProjectUserFeedback from 'app/views/projectUserFeedback';
+import ProjectUserFeedback from 'app/views/userFeedback/project';
```

Comment:
* Since we're going to eventually remove this project version - can we have them as `app/views/projectUserFeedback` and `app/views/organizationUserFeedback` to keep them consistent? (Container can probably just go in organization version)
* While we're in here can we codesplit `projectUserFeedback`?
* Should we add a route + sidebar icon to `organizationUserFeedback`?

Reply:
I'd prefer to keep all the user feedback modules grouped together since there is a lot of overlap between these - it'll be a bit easier to work on and read code all in one place.

Added codesplitting for the `projectUserFeedback` route.

We'll certainly need a route and link for `organizationUserFeedback`, I'll add it together with the implementation as a follow up to this.

Reply:
Can we at least rename the file to `projectUserFeedback` so we can still quick open using the same filename pattern?

Reply:
sure